XcodeAutoBasher/XcodeAutoBasher/VOKXcodeAutoBasher.m

```diff
@@ -69,6 +74,11 @@ - (id)initWithBundle:(NSBundle *)plugin
     return self;
 }
 
+- (void)applicationDidFinishLaunching:(NSNotification *)notification
+{
+    [self setupMenuItem];
```

Comment:
Presumably, there exist some versions of Xcode on which the existing `setupMenuItem` call was working, so won't those versions now have it run wherever that call is and then run again where you added it?